server/model/MatchModel.js

```diff
@@ -0,0 +1,15 @@
+export class MatchModel {
+  constructor(userNameArr, userIDArr) {
+
+    // variables for storing who starts the match and their unique IDs
+    this.userNameArr = userNameArr;   // array datatype
```

Comment:
Not necessary needed since you store `userIDArr`

Reply:
Changes were made.